lib/platform/bitbucket-server/__fixtures__/responses.ts

```diff
@@ -179,7 +179,7 @@ function generateServerResponses(endpoint) {
     [`${endpoint}/rest/api/1.0/projects/SOME/repos/repo/pull-requests/5/commits?withCounts=true`]: {
       GET: {
         totalCount: 1,
-        values: [ { author: { emailAddress: 'bot@renovateapp.com'} } ],
+        values: [{ author: { emailAddress: 'bot@renovateapp.com' } }],
```

Comment:
@viceice on one hand I was going to say "Revert formatting changes to fixtures" but on the other hand I wonder if this file is an indicator that prettier is picking up *.ts files within `__fixtures__` and needs reconfiguring?